websites/J/Joox/presence.ts

```diff
@@ -30,7 +31,9 @@ presence.on("UpdateData", async () => {
       presenceData: PresenceData = {
         details: title,
         state: author,
-        largeImageKey: "icon",
+        largeImageKey: cover
+          ? document.querySelector<HTMLImageElement>(`img[alt="${title}"]`).src
+          : "icon",
```

Comment:
https://discord.com/channels/493130730549805057/607524579874832446/928096461994987540

Reply:
the url is not that long (90~) so not really needed